plugins/routeros-suspension/src/main.php

```diff
@@ -1,9 +1,29 @@
 <?php
 
+use Ubnt\UcrmPluginSdk\Service\PluginConfigManager;
 use UcrmRouterOs\Service\Suspender;
 
 require_once __DIR__ . '/vendor/autoload.php';
 
 (static function () {
-    (new Suspender())->suspend();
+    $config = (new PluginConfigManager())->loadConfig();
+
+    $ipAddresses = array_map('trim', explode(PHP_EOL, $config['mikrotikIpAddress']));
+    $userNames = array_map('trim', explode(PHP_EOL, $config['mikrotikUserName']));
+    $passwords = $config['mikrotikPassword']
+        ? array_map('trim', explode(PHP_EOL, $config['mikrotikPassword']))
+        : [];
+
+    $configs = [];
+    while ($ipAddress = array_shift($ipAddresses)) {
+        $configs[] = array_merge($config, [
+            'mikrotikIpAddress' => $ipAddress,
+            'mikrotikUserName' => array_shift($userNames),
+            'mikrotikPassword' => array_shift($passwords) ?? '',
+        ]);
+    }
+
+    foreach ($configs as $config) {
+        (new Suspender($config))->suspend();
```

Comment:
I'm not sure how much info is logged about the plugin execution. Might be nice to add an info log here that the process is starting / finished for configuration XYZ.